src/ReactiveUI.AndroidSupport/ReactiveRecyclerViewViewHolder.cs

```diff
@@ -51,6 +51,16 @@ protected ReactiveRecyclerViewViewHolder(View view)
                 h => view.LongClick += h,
                 h => view.LongClick -= h)
                     .Select(_ => AdapterPosition);
+
+            SelectedWithViewModel = Observable.FromEventPattern(
```

Comment:
We are attempting to use FromEvent() these days for performance reasons. FromEventPattern is double the performance.